src/field_msg.cpp

```diff
@@ -5,7 +5,9 @@
 #include <bitset>
 #include <ctime>
 
-FieldMessage::FieldMessage()
+namespace msg{
+
+Message::Message()
     : msg_bitmap(0b11),
```

Comment:
fill bitmask zeros to the nearest multiple of eight 
```suggestion
    : msg_bitmap(0b00000011),
```
But it is better to use constants